go.mod

```diff
@@ -17,20 +17,20 @@ require (
 	github.com/go-playground/validator/v10 v10.11.1
 	github.com/gogo/protobuf v1.3.2
 	github.com/google/go-cmp v0.6.0
-	github.com/google/uuid v1.4.0
+	github.com/google/uuid v1.6.0
 	github.com/gorilla/handlers v1.4.2
 	github.com/gorilla/mux v1.8.0
-	github.com/gorilla/websocket v1.5.1
+	github.com/gorilla/websocket v1.5.3
 	github.com/hashicorp/go-multierror v1.1.1
-	github.com/hashicorp/golang-lru/v2 v2.0.5
+	github.com/hashicorp/golang-lru/v2 v2.0.7
 	github.com/ipfs/go-cid v0.4.1
 	github.com/kardianos/service v1.2.2
 	github.com/klauspost/reedsolomon v1.11.8
-	github.com/libp2p/go-libp2p v0.33.2
-	github.com/multiformats/go-multiaddr v0.12.3
-	github.com/multiformats/go-multiaddr-dns v0.3.1
+	github.com/libp2p/go-libp2p v0.38.0
```

Comment:
This increase of libp2p version would require load testing. Libp2p had regressions in earlier versions, so I would suggest to test the performance and resource consumption in integration and probably with mainnet dev cluster, if possible.